lib/dataloader/ecto.ex

```diff
@@ -57,7 +86,7 @@ if Code.ensure_loaded?(Ecto) do
 
     # as is this
     loader
-    |> Dataloader.load(:accounts, :user, :organization)
+    |> Dataloader.load(:accounts, :user, organization)
```

Comment:
It looks like this is demonstrating the simple query by association on an organization. I don't think organization was meant to be an atom there, but perhaps there is some deep magic I'm still not getting 😄